Assets/SEE/Net/Actions/Drawable/AddBlinkEffectNetAction.cs

```diff
@@ -0,0 +1,70 @@
+﻿using SEE.Controls.Actions.Drawable;
+using SEE.Game.Drawable;
+using SEE.GO;
+using SEE.Utils;
+using UnityEngine;
+
+namespace SEE.Net.Actions.Drawable
+{
+    /// <summary>
+    /// This class is responsible for add the blink effect on an object on a drawable on all clients.
+    /// </summary>
+    public class AddBlinkEffectNetAction : AbstractNetAction
```

Comment:
The network actions for drawables have a lot of redundancy that can be avoided.

I introduced a common superclass `DrawableNetAction `for network actions for drawables.
This class allows to get rid of a lot of redundant code:

- Subclasses need not implement ExecuteOnServer().
- The drawable is found by the constructor. No need to do that in the subclasses.
- Provides two methods, FindChild and TryFindChild to find a child.
- Exceptions are thrown in those methods. Subclasses do not need to deal with these.

I adjusted all classes in `SEE.Net.Actions.Drawable` from `AddBlinkEffectNetAction` to `EditLineSecondaryColorNetAction` in the alphabetic order. Please adjust the remaining classes accordingly.

Reply:
Please check also for nested if statements with the same condition and remove those. That is, I found instances of:

```cs
if (!IsRequester())
{
   if (!IsRequester())
   {
```

Reply:
I did it, but i had to change it a little bit. The problem was that the drawable object were searched in the constructor method, but this would only be called by the calling user. The drawable object search must be executed in the `ExecuteOnClient` method because the instances of the GameObjects are different between users game instances.